sklearn/calibration.py

```diff
@@ -508,7 +508,8 @@ def _compute_predictions(pred_method, X, n_classes):
     if method_name == 'decision_function':
         if predictions.ndim == 1:
             predictions = predictions[:, np.newaxis]
-    elif method_name == 'predict_proba':
+    elif method_name == 'predict_proba' or method_name == '_predict_proba':
+        # The `_predict_proba` option is needed for `VotingClassifier`
```

Comment:
I do not think special casing `VotingClassifier` is great. I think it would be better to get `_get_prediction_method` to return a tuple `(callable, method_name)` and then use the `method_name` here.

(`_get_prediction_method ` is always called before `_compute_predictions`)

Reply:
I agree. I have updated the code.